src/style.css

```diff
@@ -1698,8 +1722,19 @@ div.bad-example {
   border-color: #9e9e9e transparent transparent transparent;
 }
 
+.tooltip-text::after {
+  content: "";
+  position: absolute;
+  top: 100%;
+  left: 50%;
+  transform: translateX(-50%);
+  border-width: 5px;
+  border-style: solid;
+  border-color: #9e9e9e transparent transparent transparent;
+}
+
```

Comment:
Same here

Reply:
Done - I replaced the styling with Tailwind 😊 also added linter for better tailwind